src/libs/ReceiptUtils.ts

```diff
@@ -25,7 +30,7 @@ function getThumbnailAndImageURIs(path, filename) {
         return {thumbnail: `${path}.1024.jpg`, image: path};
     }
 
-    const {fileExtension} = FileUtils.splitExtensionFromFileName(filename);
+    const {fileExtension} = FileUtils.splitExtensionFromFileName(filename) as {fileExtension?: string};
```

Comment:
Could we extract `{fileExtension?: string}` into a separate type?

Reply:
Done